honeybee/model.py

```diff
@@ -1349,6 +1375,21 @@ def to_dict(self, included_prop=None, triangulate_sub_faces=False):
 
         return base
 
+    @staticmethod
+    def to_json(hb_dict, folder_path, name="unnamed"):
+        """Writes a Honeybee model dictionary to hbjson
+
+        Args:
+            hb_dict (dictionary): A Honeybee model in the form of a dictionary
+            folder_path (A text string): Folder location where json will be written
+            name (str, optional): File name. Defaults to "unnamed".
```

Comment:
These are not formatted correctly for sphinx and won't show up correctly in the online docs.  The argument type should come after the `:`. See all of the other docstrings in this file.  You should also probably read the contributing docs on this:
https://github.com/ladybug-tools/contributing/wiki/1.-How-to-write-docstrings-for-Ladybug-Tools